src/localization.rs

```diff
@@ -265,7 +267,7 @@ impl L10nManager {
             .current_bundle
             .format_pattern(key, value, args, &mut errs);
         for err in errs {
-            eprintln!("localization error {:?}", err);
+            error!("localization error {:?}", err);
```

Comment:
I vote `warn!`. This is tricky; but basically `error!` is the highest log level we have, and should probably be reserved for things that are fatal or otherwise compromise the execution of the application. This is maybe an error though, it's hard to say. I think that [this summary](https://stackoverflow.com/a/2031209/1529922) is a reasonable overview of how to think about the different log levels, this is definitely a 'feel' thing, though!

Reply:
Yeah. I just ventured a guess with most of them, but didn't feel familiar enough with the internals of the library in a lot of cases. Thanks for taking the time. 😄